zschema/leaves.py

```diff
@@ -411,18 +415,42 @@ class IndexedBinary(Binary):
 class DateTime(Leaf):
 
     ES_TYPE = "date"
-    BQ_TYPE = "TIMESTAMP"
+    BQ_TYPE = "DATETIME"
     EXPECTED_CLASS = [str, int, unicode]
 
     VALID = "Wed Jul  8 08:52:01 EDT 2015"
     INVALID = "Wed DNE  35 08:52:01 EDT 2015"
 
+    MIN_VALUE = "1752-01-01 00:00:00.000000"
```

Comment:
Why not `0001-01-01 00:00:00.000000`?

Reply:
Because dates before 1752 didn't have an agreed upon calendar and have no meaning.

Reply:
Wait, really?